Formula/braid.rb

```diff
@@ -0,0 +1,68 @@
+class Braid < Formula
+  desc "Simple tool to help track vendor branches in a Git repository"
+  homepage "https://cristibalan.github.io/braid/"
+  url "https://rubygems.org/gems/braid-1.0.21.gem"
+  sha256 "97300ec6fd9172b25fd05f2c88107e5129a7b85a5f63c73aea40c266b1d80508"
+  head "https://github.com/cristibalan/braid.git"
```

Comment:
The `head` spec is probably more trouble than it's worth here since we'd need to unpin the dependencies for that.